apps/antalmanac/tests/patch-notes.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import PatchNotes from '$components/PatchNotes';
+
+const setLocalStorage = (key: string, value: string) => {
+    window.localStorage.setItem(key, value);
+};
+
+describe('PatchNotes Component', () => {
+    describe('when latestPatchSeen is not equal to latestPatchNotesUpdate', () => {
+        it('should show the dialog', () => {
+            setLocalStorage('latestPatchSeen', '20230513');
+
+            const mockLatestPatchNotesUpdate = '20230819';
+            if (mockLatestPatchNotesUpdate != localStorage.getItem('latestPatchSeen')) {
+                render(<PatchNotes />);
+            }
```

Comment:
No, this shouldn't be handling any sort of rendering logic; that defeats the purpose of testing the component. The value of local storage should be manipulated, and the component should be verified to do the correct behavior depending on the value in local storage.

i.e. if we change the value in local storage to be different from the latest patch notes, then we expect no dialog to appear if the patch notes is rendered.